dist/images/ovnkube.sh

```diff
@@ -96,6 +96,7 @@ fi
 # OVN_NORTHD_BACKOFF_INTERVAL - ovn northd backoff interval in ms (default 300)
 # OVN_ENABLE_SVC_TEMPLATE_SUPPORT - enable svc template support
 # OVN_ENABLE_DNSNAMERESOLVER - enable dns name resolver support
+# OVNKUBE_OBSERV_ENABLE - enable observability for ovnkube
```

Comment:
we are not really good with the consistency here, https://github.com/ovn-org/ovn-kubernetes/blob/9030c8acbf6b6ad8e1aff1ad271374675e297672/dist/images/ovnkube.sh#L77-L81
I just prefer the same name across all files, so following `kind-helm.sh` together with `kind.sh` naming, so that we don't call the same optinon `OVN_OBSERV_ENABLE` in one file and `OVN_ENABLE_OBSERV` in another.

Reply:
Ack.